identifiers/country-mx/states.csv

```diff
@@ -0,0 +1,34 @@
+id,name
+ocd-division/country:mx,México
```

Comment:
Are there any particular reason that we use 'Mexico City' for the state name (without the accent) but 'México' for the country name (with the accent)?